python/mxnet/gluon/block.py

```diff
@@ -91,22 +92,22 @@ def create(prefix, params, hint):
     def __enter__(self):
         if self._block._empty_prefix:
             return self
-        self._old_scope = getattr(_BlockScope._current, "value", None)
+        self._local._old_scope = getattr(_BlockScope._current, "value", None)
         _BlockScope._current.value = self
```

Comment:
Shall we make `_BlockScope._current` also thread-local for consistency?

Reply:
Seems that it has been thread-local ?
https://github.com/apache/incubator-mxnet/blob/f3e8dc14de881a5b8ccc3ff5985167f8d6fa2fb8/python/mxnet/gluon/block.py#L47